pages/tutorials/pipeline_upgrade.md.erb

```diff
@@ -0,0 +1,75 @@
+1. Migrate all existing pipelines, either one at a time or using the bulk migration button
+
+After migrating to YAML Steps, you will no longer be able to access the web-based step editor.
+
+A new reference sidebar is available alongside the YAML editor. Each step type lists the available top level attributes with short descriptions, for more detailed information on each you can click through to the relevant documentation page
+
+If you've been using the YAML steps editor during its beta, none of your pipelines will have any changes applied. 
+
+### Compatibility issues
+
+Using YAML Steps changes the order in which environment variables are interpolated.
+
+There are two precedence changes: 
+1. Step-level environment variables will now take precedence over build-level variables (i.e. variables set when creating a new build)
+1. Top level `env` blocks in YAML no longer override step-level environment variables
+
+For example, in the `pipeline.yml` below, the command will echo `step` after you have migrated to using YAML Steps:
+
+```yaml
+env:
+  LEVEL: "pipeline"
+
+steps:
+  command: "echo $LEVEL"
+  env:
+    LEVEL: "step"
+```
+{: codeblock-file="pipeline.yml"}
+
+Prior to migrating to YAML Steps, the command will echo `pipeline`.
+
+## Using YAML Steps for new pipelines
+
+To use the YAML Steps editor for new pipelines created in your organization, you'll need to opt-in on the Pipeline YAML Migration page in Organization Settings. 
+
+<%= image "bulk-migration.png", width: 1744/2, height: 446/2, alt: "Screenshot of the button to use YAML steps for new pipelines" %>
+
+Clicking the button to start using YAML pipelines won't change anything for any existing pipelines in your organization. Migration of existing pipelines will need to be completed either individually or using the bulk migration tool. See [Steps to migrate existing pipelines](#steps-to-migrate-existing-pipelines) below for further information.  
+
+## Migrating existing pipelines
+
+The migration page can be accessed by organization administrators from the Organization Settings page. On this page you can see which pipelines have already been migrated and which are yet to migrate. 
+
+### Individual migration
+
+You can migrate each pipeline individually from the Pipeline Settings page. Under the web-based Steps editor is a button to Convert to YAML Steps:
+
+<%= image "individual-migration.png", width: 1730/2, height: 1086/2, alt: "Screenshot of the YAML Steps conversion panel and button" %>
+
+Click this button to convert your pipeline steps into YAML. The new YAML steps version of your pipeline will be auto-saved during migration, and will replace the web-based steps.
+
+If you expect to have compatibility issues with your environment variables, individual migration is recommended. 
+
+### Bulk migration
+
+You can migrate all of the pipelines in your organization at the same time using the bulk migration button on the migration page.
+
```

Comment:
<section class="Docs__note">
<p>If you expect to have < a href="#compatibility-issues">compatibility issues</a> with your environment variables, migrate your pipelines < a href="#individually">individually</a>. 
</p>
</section>